mmpose/models/losses/regression_loss.py

```diff
@@ -20,18 +29,72 @@ def forward(self, output, target, target_weight):
         Note:
             batch_size: N
             num_keypoints: K
+            dimension of keypoints: D (D=2 or D=3)
+
+        Args:
+            output (torch.Tensor[N, K, D]): Output regression.
+            target (torch.Tensor[N, K, D]): Target regression.
+            target_weight (torch.Tensor[N, K, D]):
+                Weights across different joint types.
+        """
+        if self.use_target_weight:
+            loss = self.criterion(output * target_weight,
+                                  target * target_weight)
+        else:
+            loss = self.criterion(output, target)
+
+        return loss * self.loss_weight
```

Comment:
Looks like that the implementation of `SmoothL1Loss` is changed.

Why remove the ` / num_joints` part?

Reply:
Yes. 
In nn.SmoothL1Loss, reduction='mean', so the loss has already been divided by the number of elements in the output. No need to divide `num_joints` again.

Reply:
Okay, does this change affect any ckpt?

Reply:
Yes. This may affect the ckpts of deeppose.

Reply:
I retrained the model, but did not notice significant difference in accuracy.